src/test/java/leave_room/LeaveRoomInteractorTest.java

```diff
@@ -0,0 +1,50 @@
+package leave_room;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import database.*;
+import models.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import static org.mockito.Mockito.*;
+
+public class LeaveRoomInteractorTest {
+    private LeaveRoomInteractor leaveRoomInteractor;
+    private RoomDBGateway roomDBGatewayMock;
+    private LeaveRoomOutputBoundary leaveRoomOutputBoundaryMock;
+
+    @BeforeEach
+    public void setup() {
+        roomDBGatewayMock = Mockito.mock(RoomDBGateway.class);
+        leaveRoomOutputBoundaryMock = Mockito.mock(LeaveRoomOutputBoundary.class);
+        leaveRoomInteractor = new LeaveRoomInteractor(roomDBGatewayMock, leaveRoomOutputBoundaryMock);
+    }
+
+    @Test
+    public void testLeaveRoomSuccessfully() {
+        int roomID = 1;
+        int userID = 111;
+        int userID2 = 222;
```

Comment:
Maybe it's worth putting these in setup since you use them twice? Up to you.

Reply:
see 0c02df6